tests/test_manchester.py

```diff
@@ -0,0 +1,39 @@
+from ontopy import get_ontology
+from ontopy.manchester import evaluate
+from owlready2 import And, Or, Not, Inverse
+
+
+def check(s, expected):
+    r = evaluate(emmo, s)
+    print(s, '-->', r)
+    assert repr(r) == repr(expected)
+
+
+emmo = get_ontology().load()
+
+check('Item', emmo.Item)
+check('not Item', Not(emmo.Item))
+check('not not Item', emmo.Item)
+check('not (not Item)', Not(Not(emmo.Item)))
+check('hasPart some Atom', emmo.hasPart.some(emmo.Atom))
+check('Atom and not Molecule', emmo.Atom & Not(emmo.Molecule))
```

Comment:
This should be added in a `test_xxx`-function in the same way as the other tests.

Reply:
I have done it now, but is the benefit of it? It just makes quick testing during implementation more cumbersome, since I don't have access to the local scope within the test-function.

Reply:
I agree with you on this. Casper explained it to me once, but it didn't stick. @CasperWA, could you please try again?

Reply:
The pytest system works by instantiating a local environment and running all the `test_...` functions and `Test...` classes it can find under the specified path. Creating a `test_...` function properly defines your test, make it skippable/togglable and allows for you to bring in special variables called "fixtures" in the pytest system. All of this is useful to set up a manageable unit testing suite and it should, in theory, also ensure that you dedicate a test function to testing a specific aspect of the code.

In essence, mixing the different tests into a single test function makes it difficult to separate the tests from each other and pinpoint an issue should the test fail.
And putting tests into functions allows the pytest system to recognize it properly as a test, making one able to utilize the full suite of options the pytest system offers.